modules/apps/oauth2-provider/oauth2-provider-api/src/main/java/com/liferay/oauth2/provider/model/OAuth2ApplicationModel.java

```diff
@@ -36,7 +36,7 @@
  */
 @ProviderType
 public interface OAuth2ApplicationModel
-	extends AuditedModel, BaseModel<OAuth2Application>, ShardedModel {
+	extends BaseModel<OAuth2Application>, ShardedModel, StagedAuditedModel {
```

Comment:
@rotty3000  The major bump is caused by this change in interfaces made by SB buildService. I don't understand why this would be considered a major version change because `StagedAuditedModel` extends `AuditedModel`. Any thoughts?

Reply:
Got it. I think this is a flaw in bnd's baseline analysis. That should be a compatible change. I believe I've seen that type of miscalculation before. Let me see if we have a workaround we can use.